keyboards/yetis/rules.mk

```diff
@@ -0,0 +1,2 @@
+SERIAL_DRIVER = vendor
+CONVERT_TO = rp2040_ce
```

Comment:
This is invalid.
```suggestion
```

Reply:
removed all as it was moved to `info.json`